libs/priceactionlab/PriceActionLabSystem.cpp

```diff
@@ -254,33 +254,25 @@ PALPatternPtr
 SmallestVolatilityTieBreaker::getTieBreakerPattern(PALPatternPtr pattern1, 
 						   PALPatternPtr pattern2) const
 {
-  decimal7 stop1 = pattern1->getStopLossAsDecimal();
-  decimal7 stop2 = pattern2->getStopLossAsDecimal();
-  decimal7 target1 = pattern1->getProfitTargetAsDecimal();
-  decimal7 target2 = pattern2->getProfitTargetAsDecimal();
-
-  if ((stop1 == stop2) && (target1 == target2))
-    {
-      //      std::cout << "getTiebreakerPattern: stop and profit target are equal" << std::endl;
+  auto volRank = [](const PALPatternPtr &p){
+        if (p->isLowVolatilityPattern())       return 0;
```

Comment:
Might be clearer to have an `enum` here?

Reply:
Can you show an example of what you are thinking of?

Reply:
Something like:
```
enum class VolatilityPattern : int {
  LOW = 0,
  NORMAL= 1,
  HIGH = 2,
  VERY_HIGH = 3,
  NONE = 4
}
```
And then we would have a simple function like:
```
p->VolatilityPattern() -> VolatilityPattern
```

Reply:
That does not seem any clearer to me.